StereoKit/Assets/Anchor.cs

```diff
@@ -15,7 +16,17 @@ public string Id
 		public BtnState Tracked => NativeAPI.anchor_get_tracked(_inst);
 		public bool Persistent => NativeAPI.anchor_get_persistent(_inst);
 		public string Name => NativeHelper.FromUtf8(NativeAPI.anchor_get_name(_inst));
-
+		public bool TryGetPerceptionAnchor<T>(out T spatialAnchor) where T : class
```

Comment:
I imagine this is expecting a pretty specific Type here? Thanks for using generics to avoid the library coupling! Could we also get commentdocs that indicate specifically what Type(s) should work here?

Reply:
Added! I think we're limited to the "class" constraint without a reference to winmd. I also figured to specify an instance of the SpatialAnchor type instead of its `ISpatialAnchor, ISpatialAnchor2` implementation since they're internal